### PR TITLE
Added Table component and translations

### PR DESCRIPTION
**Pull request by Maxim-Lam**
### Description

- added table component and translations for it

#### List of proposed changes:

- see description
-

### What to test for/How to test

Add the following snippet to line 55 of BenefitCard.tsx,

`<div>
      <Table
        data={[
          { age: 52, amount: 65000 },
          { age: 53, amount: 75000 },
          { age: 54, amount: 78000 },
          { age: 55, amount: 80000 },
          { age: 56, amount: 85000 },
        ]}
      />
</div>`
navigate to results page and scroll down to the benefit card.

Switch from en to fr

make sure the table works on mobile screens

### Additional Notes

I don't know how we are planning on sending the data to the component so this is how I coded in the mean time.

### CODE REVIEW

**Comment by MarcoGoC** [comment]
Don't merge until previous PR are approved, this is to have a baseline for Prod before introducing these changes

File: components/Table.tsx
Comment (MarcoGoC) [inline components/Table.tsx]: see numberToStringCurrency function

File: components/Table.tsx
Comment (MarcoGoC) [inline components/Table.tsx:26]: use scope  I know it is not necessary but a similar table was rejected

**Review by alex-solo — APPROVED** [review]

**Pull request merged**
